Build error on Java 11

I get a Gradle error when my Java home is JDK11. No issue when building with JDK8.

```
./gradlew check

> Configure project :wire-runtime
Kotlin Multiplatform Projects are an experimental feature.
Target 'winX64' for platform mingw_x64 is ignored during build on this macos_x64 machine. You can build it with a mingw_x64 host.

> Configure project :wire-tests
Target 'winX64' for platform mingw_x64 is ignored during build on this macos_x64 machine. You can build it with a mingw_x64 host.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':wire-tests:compileTestJava'.
> Could not resolve all task dependencies for configuration ':wire-tests:testCompileClasspath'.
   > Could not resolve project :wire-grpc-client.
     Required by:
         project :wire-tests
      > Cannot choose between the following variants of project :wire-grpc-client:
          - compile
          - default
          - runtime
          - testCompile
          - testRuntime
        All of them match the consumer attributes:
          - Variant 'compile' capability com.squareup.wire:wire-grpc-client:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-grpc-client' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'default' capability com.squareup.wire:wire-grpc-client:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-grpc-client' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'runtime' capability com.squareup.wire:wire-grpc-client:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-grpc-client' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'testCompile' capability com.squareup.wire:wire-grpc-client:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-grpc-client' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'testRuntime' capability com.squareup.wire:wire-grpc-client:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-grpc-client' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
   > Could not resolve project :wire-test-utils.
     Required by:
         project :wire-tests
      > Cannot choose between the following variants of project :wire-test-utils:
          - compile
          - default
          - runtime
          - testCompile
          - testRuntime
        All of them match the consumer attributes:
          - Variant 'compile' capability com.squareup.wire:wire-test-utils:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-test-utils' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'default' capability com.squareup.wire:wire-test-utils:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-test-utils' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'runtime' capability com.squareup.wire:wire-test-utils:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-test-utils' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'testCompile' capability com.squareup.wire:wire-test-utils:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-test-utils' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.
          - Variant 'testRuntime' capability com.squareup.wire:wire-test-utils:3.0.0-SNAPSHOT:
              - Unmatched attributes:
                  - Required org.gradle.dependency.bundling 'external' but no value provided.
                  - Required org.gradle.jvm.version '8' but no value provided.
                  - Required org.gradle.usage 'java-api' but no value provided.
                  - Found org.jetbrains.kotlin.localToProject 'local to :wire-test-utils' but wasn't required.
                  - Found org.jetbrains.kotlin.platform.type 'jvm' but wasn't required.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

--- a/wire-tests/jvm.gradle
+++ b/wire-tests/jvm.gradle
@@ -1,9 +1,6 @@
 apply plugin: 'java-library'
 
 kotlin.targets.matching { it.platformType.name == 'jvm' }.all { target ->
-  target.project.sourceCompatibility = JavaVersion.VERSION_1_8
-  target.project.targetCompatibility = JavaVersion.VERSION_1_8
-  
   target.project.sourceSets.test.java.srcDirs += 'src/jvmTest/proto-java'
 
   tasks['jar'].configure {

